src/Adherent/AdherentRoleEnum.php

```diff
@@ -29,4 +29,6 @@ class AdherentRoleEnum extends Enum
     public const LAREM = 'gq_ideas';
 
     public const ELECTED = 'elected';
+
+    public const MUNICIPAL_CHIEF = 'Chef municipal';
```

Comment:
this should be a value in snake_case like the others

Reply:
Wouldn't it disturb admin users to see snake_case string in a label?